pkg/controllermanager/controller/shoot/shoot_care_control.go

```diff
@@ -120,6 +120,11 @@ func (c *defaultCareControl) Care(shootObj *gardenv1beta1.Shoot, key string) err
 	)
 	shootLogger.Debugf("[SHOOT CARE] %s", key)
 
+	// if shoot has not been scheduled, requeue
+	if shoot.Spec.Cloud.Seed == nil {
```

Comment:
Nice idea! I'd however recommend moving this into `reconcileShootCareKey` to not litter the `Care` function with this kind of checks.